packages/devtools_app/lib/src/screens/network/network_screen.dart

```diff
@@ -112,7 +110,8 @@ Example queries:
 
 class _NetworkScreenBodyState extends State<NetworkScreenBody>
     with AutoDisposeMixin {
-  NetworkController _networkController;
+  bool initialized = false;
```

Comment:
Is there reason for initialized to be public?

Reply:
Nope, I'll make it private.